jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/execution/EcsCloudDebugRunConfiguration.kt

```diff
@@ -64,7 +64,7 @@ class EcsCloudDebugRunConfiguration(project: Project, private val configFactory:
 
     override fun getState(executor: Executor, environment: ExecutionEnvironment): CloudDebugRunState? =
         try {
-            val cloudDebugRunSettings = resolveEcsServiceCloudDebuggingRunSettings()
+            val cloudDebugRunSettings = resolveEcsServiceCloudDebuggingRunSettings(true)
```

Comment:
Not sure I follow.

Ideally it shouldn't need to do any validation, but we conflated validation with building the immutable state. So I guess it could be shallow since it should be guarded by checkConfigBeforeRun()

Reply:
as long as that doesn't affect the cascading drop-down loads.

Reply:
Switched it to false, but also that loads data in the background so yay